data/hai/hai reveal 5 philosophers.txt

```diff
@@ -1011,84 +1011,129 @@ mission "Hai Reveal [B13] Freedom"
+			`	"The pirates must defend this planet against a space-based lifeform that attacks in flocks. Our ships gave them a significant advantage. Fortunately, we have one more left to defend against any threats." The Korath eyes the Republic operatives as he waits for the translator to catch up.`
 			`	"I must ask you keep the presence of Korath here a secret. If the Archons or other Exiles hear of this place, they may attack to retrieve us, or the Exiles may simply pillage. We will remain here in hiding unless the Exiles finally learn to settle on their planet."`
 				goto questions
 			label "translation services"
-			`	The Korath leader hisses and replies while the Hai translates. "No, we cannot. Few among the Korath know your language, and we must remain here in secret. Perhaps in time there will be a way, but not yet."`
+			action
+				set "Hai Reveal [B13] Freedom: label translation services"
+			`	The Korath leader hisses. "No, we cannot. We must remain here in secret. Perhaps in time there will be a way, but not yet."`
 				goto questions
 			label "cadmium"
+			action
+				set "Hai Reveal [B13] Freedom: label cadmium"
 			`	The Korath leader looks to the Hai, who responds, "Is that how you were led back here?"`
 			`	"Indirectly," you reply.`
-			`	The Hai leader continues, "Fascinating, we always wondered if it would pique someone's attention! Our Korath friends here brought a ship with some remarkable technology they call a "Systems Core". None of the surviving Korath who got here had the knowledge to replicate it exactly, but they were asked to try and many of them seem to have an uncanny knack for engineering. With their help, we managed to build some of our own in a fashion. Being not quite right, they work, but they fuse irrevocably with the ship, and some flaw in their function causes cadmium particulate, which is shed by the system maintenance processes, into the air where it is recirculated and ignored by the system intended to filter it out, since it's part of it."`
+			`	The Hai leader continues, "Fascinating, we always wondered if it would pique someone's attention! Our Korath friends here brought a ship with remarkable technology called a "Systems Core". None of the surviving Korath had the knowledge to replicate it, but they were asked to try and many of them have an uncanny knack for engineering. With their help, we built some of our own in a fashion. Being not quite right, they work, but they fuse irrevocably with the ship, and some flaw in their function causes cadmium particulate, which is shed by the system maintenance processes, into the air where it is recirculated and ignored by the system intended to filter it out, since it's part of it."`
```

Comment:
```suggestion
			`	The Hai leader continues, "Fascinating, we always wondered if it would pique someone's attention! Our Korath friends have ships equipped with these remarkable "Systems Cores", which are apparently vital to all Korath ships. None of the surviving Korath had the knowledge to replicate it, but they were asked to try and many of them have an uncanny knack for engineering. With their help, we built some of our own in a fashion. Being not quite right, they work, but they fuse irrevocably with the ship, and some flaw in their function causes cadmium particulate. The cadmium is shed by the system maintenance processes into the air, where it is recirculated, and ignored by the system intended to filter it out since it's a part of it."`
```
That last sentence was hard to speak out.